hack/00-osd-managed-prometheus-exporter-ebs-iops-reporter.selectorsyncset.yaml.tmpl

```diff
@@ -266,7 +270,9 @@ objects:
               - /secrets/aws/credentials.ini
               - -c
               - /config/env/CLUSTERID
-              image: quay.io/openshift-sre/managed-prometheus-exporter-initcontainer:v0.1.9-2019-03-28-4e558131
+              - -m
+              - /config/env/models/cloudwatch/2010-08-01/service-2.json
```

Comment:
how is this path(and the json) being generated? Nice work in general @robotmaxtron :)

Reply:
The path is being generated and the json included from the init container that runs ahead of this "main" container. See: https://github.com/openshift/managed-prometheus-exporter-initcontainer/pull/19